mplviz.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+from symbols import rr, rf, y, qs, ps
+from model import N, o, p, rear_wheel, front_wheel
+from symmeplot.matplotlib import Scene3D
+from base_simulation import p_arr, res
+
+(times, q_traj, u_traj, slip_traj, f_traj, fz_traj, con_traj, q9_traj,
+ q10_traj, r_traj) = res
+
+scene = Scene3D(N, o, scale=0.5)
+
+scene.add_frame(N, p)  # kick plate
+
+rear_wheel_plot = scene.add_body(rear_wheel)
+rear_wheel_plot.attach_circle(
+    rear_wheel.masscenter,
+    rr,
+    rear_wheel.frame.y,
+    facecolor="red", edgecolor="k")
+
+front_wheel_plot = scene.add_body(front_wheel)
+front_wheel_plot.attach_circle(
+    front_wheel.masscenter,
+    rf,
+    front_wheel.frame.y,
+    facecolor="red", edgecolor="k")
+
+#scene.add_body(rear_frame_body)
+#scene.add_line(some_points_describing_the_rear_frame)
+#
+#scene.add_body(front_frame_body)
+#scene.add_line(some_points_describing_the_front_frame)
+
+# For a single state plot:
+scene.lambdify_system(qs + [y] + list(ps))
+scene.evaluate_system(*np.hstack((q_traj[0, :], r_traj[0, 4:5], p_arr)))
+
```

Comment:
```suggestion
scene.plot()

```

Reply:
This is confusing, I should adjust the implementation of `symmeplot` to only actually do things like attaching hover when you run `plot` and `plot` should throw an error when you try to run it a second time.

Reply:
Woohoo!

![image](https://github.com/user-attachments/assets/1e61a0b7-a50f-4a00-ad99-45c8a0dca676)